R/stat_stepribbon.R

```diff
@@ -0,0 +1,77 @@
+#' Step ribbon statistic
+#'
+#' Provides stairstep values for ribbon plots
+#'
+#' @name stat_stepribbon
+#' @inheritParams ggplot2::geom_ribbon
+#' @param geom which geom to use; defaults to "`ribbon`"
+#' @param direction `hv` for horizontal-vertical steps, `vh` for
+#'   vertical-horizontal steps
+#' @references [https://groups.google.com/forum/?fromgroups=#!topic/ggplot2/9cFWHaH1CPs]()
+#' @return a ggplot
+#' @examples
+#' library(ggplot2)
+#'
+#' survival::survfit(survival::Surv(time, status) ~ 1, data = survival::lung) %>%
```

Comment:
#' survival::survfit(survival::Surv(time, status) ~ 1, data = survival::lung) %>%
#'   survival::survfit0() %>%
#'   broom::tidy() %>%
#'   ggplot2::ggplot(ggplot2::aes(x = time, y = estimate, ymin = conf.low, ymax = conf.high)) +
#'   ggplot2::geom_step() +
#'   ggplot2::geom_ribbon(stat = "stepribbon", alpha = 0.2)

Reply:
we are prefixing all examples and no library calls in examples

Reply:
updated